nasademo/src/main/resources/db/migration/V1_addCameraTable.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS cameras(
+    id BIGINT not null,
+    nasa_id BIGINT,
```

Comment:
I suppose all other colums should have not null restriction and nasa_id should have unique constraint